packaging/package.sh

```diff
@@ -67,22 +75,32 @@ function packaging() {
             exit 2
         fi
     fi
+
     VERSION=$(cd ../; $MVN org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep --color=none '^[0-9]\.')
+    BASEVER=$(echo "$VERSION" | sed 's/-SNAPSHOT//g')
```

Comment:
-1 we would want to keep the SNAPSHOT in the package names.

Reply:
on this line I'm trying to get BRANDING string and not removing SNAPSHOT from final package name. That's don in line 94 and 97 (`$(date +%s)` instead of `SNAPSHOT`).
And I believe this will be a good change because it will prevent accidental artifacts overridden on top of each other.